packages/flutterfire_cli/lib/src/commands/config.dart

```diff
@@ -257,7 +257,20 @@ class ConfigCommand extends FlutterFireCommand {
   }
 
   String? get relativeMacosServiceFilePath {
```

Comment:
Should we rename this to FolderPath? Since we cannot change the fileName?

Reply:
I meant if we add conditions to make sure that the filename is always 'GoogleService-Info.plist', should we just let the user give the `relativeMacosServiceFolderPath` so we can detect automatically the 'GoogleService-Info.plist'?

Reply:
actually, I've just looked at this and we should be able to just make it one getter rather than two, I thought it was used elsewhere in the program 👍